src/Workspaces/Core/Portable/ExtensionManager/IExtensionManagerExtensions.cs

```diff
@@ -84,7 +86,7 @@ public static void PerformAction(this IExtensionManager extensionManager, object
         public static async Task<T> PerformFunctionAsync<T>(
             this IExtensionManager extensionManager,
             object extension,
-            Func<Task<T>> function,
+            Func<Task<T>?> function,
```

Comment:
Missed the equivalent annotation above in PerformActionAsync that is just Task returning.